src/prommis/precipitate/precipitate_liquid_properties.py

```diff
@@ -148,6 +207,23 @@ def build(self):
             ]
         )
 
+        self.split_elements = Set(
```

Comment:
A doc string or comment would be good here to explain why these were selected (my guess is due to data available in Mintec but it would be good to be clear.).